adafruit_displayio_sh1107.py

```diff
@@ -141,11 +141,11 @@ class SH1107(displayio.Display):
 
     def __init__(
         self,
-        bus,
-        display_offset=DISPLAY_OFFSET_ADAFRUIT_FEATHERWING_OLED_4650,
-        rotation=0,
+        bus: displayio.I2CDisplay,
```

Comment:
Based on the `displatio_sh1107_mono_128x128_test.py`, this can also be `displayio.FourWire`.